test/java/seedu/addressbook/logic/LogicTest.java

```diff
@@ -3,7 +3,7 @@
 
 import org.junit.Before;
 import org.junit.Rule;
-import org.junit.Test;
+import org.testng.annotations.Test;
```

Comment:
You have attempted to import another third-party testing framework, but where is your actual test for the module enhancement that you have added?